types/lib/debug.d.ts

```diff
@@ -0,0 +1,64 @@
+import Class from "abitbol";
+import type CallbackManager from "./callback-manager";
+
+// TODO throw errors for console.error()
```

Comment:
what should we do ?

Reply:
https://github.com/wanadev/obsidian-core/pull/112/commits/49b759b20f9de1a744662002f63fc2460f559e05